### PR TITLE
Update chemistry example

### PR DESCRIPTION
**Pull request by guenp**
* Generate caffeine Broombridge file using OpenMolcas to support active space calculation
* Add cells that run VQE

### CODE REVIEW

File: examples/chemistry/Molecule.ipynb [hunk 1]
```diff
@@ -345,47 +390,325 @@
...
+   "metadata": {},
+   "source": [
+    "#### Run RPE"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Compare to FCI energy = -627.6287613260"
```
Comment (dgwvfxn) [inline examples/chemistry/Molecule.ipynb:502]: FCI energy is -627.63095945558848, the value you give here is the SCF energy.

File: examples/chemistry/Molecule.ipynb [hunk 2]
```diff
@@ -345,47 +390,325 @@
...
+     "output_type": "stream",
+     "text": [
+      "CPU times: user 0 ns, sys: 21.8 ms, total: 21.8 ms\n",
+      "Wall time: 38 s\n"
+     ]
+    },
+    {
+     "data": {
+      "text/plain": [
+       "(-0.3383813350711708, -627.6309264967134)"
```
Comment (dgwvfxn) [inline examples/chemistry/Molecule.ipynb:521]: This QPE energy is very close to the FCI energy (see previous comment)

File: examples/chemistry/Molecule.ipynb [hunk 3]
```diff
@@ -345,47 +390,325 @@
...
+      ]
+     },
+     "execution_count": 18,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "GetEnergyVQE.estimate_resources(\n",
```
Comment (dgwvfxn) [inline examples/chemistry/Molecule.ipynb:482]: I suggest to include the ansatz we use in the comment line

**Review by dgwvfxn — CHANGES_REQUESTED** [review]

**Review by dgwvfxn — APPROVED** [review]

**Review by dgwvfxn — APPROVED** [review]

**Review by vxfield — APPROVED** [review]
Looks good to me

**Pull request merged**